test/lib/jdk/test/whitebox/WhiteBox.java

```diff
@@ -605,6 +606,7 @@ public Object getMethodOption(Executable method, String name) {
 
   // Handshakes
   public native int handshakeWalkStack(Thread t, boolean all_threads);
+  public native boolean handshakeReadMonitors(Thread t);
```

Comment:
Why is this function added?

Reply:
We have two WhiteBox classes: sun.hotspot.WhiteBox and jdk.test.whitebox.WhiteBox. They should have the exact same contents, but handshakeReadMonitors is missing from jdk.test.whitebox.WhiteBox. So I added it since I am changing this file.

https://bugs.openjdk.java.net/browse/JDK-8271707 will remove sun.hotspot.WhiteBox so we won't have this duplication anymore.